tests/pytorch/test_models_classification_pt.py

```diff
@@ -46,9 +46,8 @@ def test_classification_architectures(arch_name, input_shape, output_size):
     # Model
     model = classification.__dict__[arch_name](pretrained=True).eval()
     _test_classification(model, input_shape, output_size)
-    # test pretrained model with different num_classes
-    model = classification.__dict__[arch_name](pretrained=True, num_classes=108).eval()
-    _test_classification(model, input_shape, output_size=(108,))
+    # Check that you can pretrained everything up until the last layer
```

Comment:
maybe better: Test whether a pretrained model can be initialized down to the last layer ? :)

Reply:
Not sure I understand?

Compared to previous features, my idea is that we can already assume:
- models are built correctly
- pretrained with default output classes is working

What needs testing here is that the checkpoint can be loaded on a model with different num_classes. So perhaps we could check that the bias of a given layer is indeed the one of the state_dict, but I don't see much more :) 

(and I try to keep unittests from running slower)

Reply:
@frgfm sorry I just mean to change this comment :
```
# Check that you can pretrained everything up until the last layer
maybe better / clearer:
# Test whether a pretrained model can be initialized down to the last layer 
```
:sweat_smile:

Reply:
@frgfm and about the unitests i think my tests from: https://github.com/mindee/doctr/pull/892/files are currently the biggest slowdown but i think we need to test each model